build.rs

```diff
@@ -48,7 +48,7 @@ fn main() {
 
         match rustc_minor_version() {
             Some(minor_ver) if minor_ver < WIN7_INTRODUCED_MINOR_VER => {
-                println!("cargo:rustc-cfg=getrandom_windows_legacy");
+                println!("cargo:rustc-cfg=getrandom_backend=\"windows_legacy\"");
```

Comment:
One thing that concerned me about this line is if it would override users who used `--getrandom_backend="whatever"` on Windows when using an older compiler.

Turns out, I can be an issue, but depends on the value of `"whatever"`.  Opened #728 to fix.